symphony-bdk-examples/bdk-spring-boot-example/src/main/resources/application.yaml

```diff
@@ -0,0 +1,15 @@
+domain: devx1.symphony.com
+
+bdk:
+  pod:
+    host: ${domain}
```

Comment:
if it is often the same value, maybe we could introduce a host property at the top level (bk.host) to avoid configuring it multiple times

Reply:
Added to the wish list : https://github.com/SymphonyPlatformSolutions/symphony-api-client-java/pull/186/files#diff-a1bfcc07b0530514177208ab2e665f35R1. 

> Do you keep a wish list somewhere for the BDK where we could track ideas and improvements?
WDYT of using this format?